examples/hello-world/step-by-step/cifar10/code/readme.md

```diff
@@ -0,0 +1,84 @@
+
+# pytorch example with SAG workflow
+
+## Code example
+
+the CIFAR10 example with pytorch is adapted from [pytorch tutorial example](https://github.com/pytorch/tutorials/blob/main/beginner_source/blitz/cifar10_tutorial.py)
+
+ 
+
+```
+
+def define_parser():
```

Comment:
Why we put this code segment in readme without explanation?

We can just link to the code directly.